CHANGELOG.md

```diff
@@ -12,7 +12,7 @@
 * **Better debug output related to sentence counting internals**. Clarify message when sentence is not counted toward sentence count because there aren't enough words, relevant for developers focused on `unstructured`s NLP internals.
 * **Faster ocr_only speed for partitioning PDF and images.** Use `unstructured_pytesseract.run_and_get_multiple_output` function to reduce the number of calls to `tesseract` by half when partitioning pdf or image with `tesseract`
 * **Adds data source properties to fsspec connectors** These properties (date_created, date_modified, version, source_url, record_locator) are written to element metadata during ingest, mapping elements to information about the document source from which they derive.
-* **Adds data source properties (date_created, date_modified, version, exists, source_url, record_locator) to Onedrive, Slack and Reddit connectors.** Implements `update_source_metadata` method to onedrive, reddit and slack connectors. 
+* **Adds data source properties to connectors** These properties (date_created, date_modified, version, source_url, record_locator) are written to element metadata during ingest, mapping elements to information about the document source from which they derive.
```

Comment:
why does it matter?

Reply:
sorry, i mean bullet points should cover "what was broken? what was fixed? why does it matter?" for enhancements, only the last part should be answered